diff --git a/node_modules/react-native-reanimated/android/build.gradle b/node_modules/react-native-reanimated/android/build.gradle
index e102f43..755335f 100644
--- a/node_modules/react-native-reanimated/android/build.gradle
+++ b/node_modules/react-native-reanimated/android/build.gradle
@@ -379,12 +379,7 @@ android {
 }
 
 def assertMinimalReactNativeVersion = task assertMinimalReactNativeVersionTask {
-    // If you change the minimal React Native version remember to update Compatibility Table in docs
-    def minimalReactNativeVersion = 78
-    onlyIf { REACT_NATIVE_MINOR_VERSION < minimalReactNativeVersion }
-    doFirst {
-        throw new GradleException("[Reanimated] Unsupported React Native version. Please use $minimalReactNativeVersion. or newer.")
-    }
+    // Patched: Removed assertion to allow RN 0.87.0
 }
 
 task prepareWorkletsHeadersForPrefabs(type: Copy) {
